portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.servlets;
```

Comment:
How did you rename this file? It looks like you're losing the version history.

Reply:
I renamed it by right-click in cloudshell. Didn't think about that, any ideas on what to do there? All of the old code was deleted anyway so it is pretty much a new file

Reply:
git-mv: https://git-scm.com/docs/git-mv

Reply:
It's useful to to go through the practice of reverting changes, which this gives you a nice opportunity to try out. Otherwise, I'd also say it's ok to proceed as is.